.github/workflows/ci.yml

```diff
@@ -319,10 +304,8 @@ jobs:
     needs:
     - lint-styles
     - lint-js
-    - build-ubuntu-node-16
-    - build-windows-node-16
-    - build-ubuntu-node-18
-    - build-windows-node-18
+    - build-windows-node-20
```

Comment:
Do we think there's value in testing in node 18 as well, or are we happy just testing in node 20 now?

Reply:
After speaking with @xander-marjoram and @jamieomaguire, I think we'll look to move these other builds to PIE Aperture, to ensure our components work with older versions of Node 👍🏼

Reply:
Happy to do this for Node 16. I think the danger with doing that for 18 too is that we're likely one of the first teams to update our stack to use that, so I think we should keep the node 18 checks it for now until we have checks elsewhere that cover that gap